src/com_tjvendors/admin/tables/vendor.php

```diff
@@ -129,6 +129,13 @@ public function check()
 		$db = Factory::getDbo();
 		$this->alias = trim($this->alias);
 
+		if (empty(trim($this->vendor_title)))
```

Comment:
you can blankspace rule from TJ library models/rules for this, and use that in xml